api/client/image/build.go

```diff
@@ -211,6 +227,9 @@ func runBuild(dockerCli *client.DockerCli, options buildOptions) error {
 
 	// Setup an upload progress bar
 	progressOutput := streamformatter.NewStreamFormatter().NewProgressOutput(progBuff, true)
+	if !dockerCli.IsTerminalOut() {
```

Comment:
Sorry, did I read it wrong? (can't see old diff anymore). I might have missed that `progressOutput` was reused.